webapp/components/test/interop.html

```diff
@@ -86,116 +48,29 @@
         });
       });
 
-      // TODO(Hexcles): Re-enable this test.
-      suite.skip('static get properties()', () => {
-        test('passRateMetadata', () => {
```

Comment:
Note that I removed these tests because "test that I do have a property name foo with a type bar" is a meaningless test; it is, of course, going to be true until something changes (such as the type or existence), and you're getting no assurances out of it (except that Polymer is doing what it says it does).